primitives/xcm/src/ethereum_xcm.rs

```diff
@@ -223,65 +223,57 @@ mod tests {
 	#[test]
 	fn test_into_ethereum_tx_with_auto_fee_v1() {
 		let xcm_transaction = EthereumXcmTransactionV1 {
-			gas_limit: U256::from(1),
+			gas_limit: U256::one(),
 			fee_payment: EthereumXcmFee::Auto,
 			action: TransactionAction::Call(H160::default()),
-			value: U256::from(0),
+			value: U256::zero(),
 			input: vec![1u8],
 			access_list: None,
 		};
-		let nonce = U256::from(0);
-		let base_fee = U256::from(1);
+		let nonce = U256::zero();
 		let expected_tx = Some(TransactionV2::EIP1559(EIP1559Transaction {
 			chain_id: 0,
 			nonce,
-			max_fee_per_gas: base_fee,
+			max_fee_per_gas: U256::zero(),
 			max_priority_fee_per_gas: U256::zero(),
-			gas_limit: U256::from(1),
+			gas_limit: U256::one(),
 			action: TransactionAction::Call(H160::default()),
-			value: U256::from(0),
+			value: U256::zero(),
 			input: vec![1u8],
 			access_list: vec![],
 			odd_y_parity: true,
 			r: H256::from_low_u64_be(1u64),
 			s: H256::from_low_u64_be(1u64),
 		}));
 
-		assert_eq!(
-			xcm_transaction.into_transaction_v2(base_fee, nonce),
-			expected_tx
-		);
+		assert_eq!(xcm_transaction.into_transaction_v2(nonce), expected_tx);
 	}
 
 	#[test]
 	fn test_legacy_v1() {
 		let xcm_transaction = EthereumXcmTransactionV1 {
-			gas_limit: U256::from(1),
+			gas_limit: U256::one(),
 			fee_payment: EthereumXcmFee::Manual(ManualEthereumXcmFee {
-				gas_price: Some(U256::from(1)),
+				gas_price: Some(U256::one()),
 				max_fee_per_gas: None,
 			}),
 			action: TransactionAction::Call(H160::default()),
-			value: U256::from(0),
+			value: U256::zero(),
 			input: vec![1u8],
 			access_list: None,
 		};
-		let nonce = U256::from(0);
-		let gas_price = U256::from(1);
+		let nonce = U256::zero();
 		let expected_tx = Some(TransactionV2::Legacy(LegacyTransaction {
 			nonce,
-			gas_price,
-			gas_limit: U256::from(1),
+			gas_price: U256::zero(),
```

Comment:
```suggestion
			gas_price: U256::one(),
```
because gas_price = U256::from(1) before or was there a reason it should be changed from 1 to 0 here?

Reply:
Previously (in V1) we double charged the fee - in the xcm executor side and in the evm - and the value provided by the user was part of the transaction. Now that we are in V2, fees are charged once - in the xcm executor, so I [removed that parameter](https://github.com/PureStake/moonbeam/pull/1790/files/cd220d965e24d6607a42a1df719c0e42f7a8e630#diff-39621bab15226365121c5dbe94941be8b77994bcae712caabb6d36b34d0b50a1R107-R125) and we always use zero.

Reply:
Thanks for explaining that